src/python/bot/tasks/task_creation.py

```diff
@@ -142,13 +143,23 @@ def create_regression_task_if_needed(testcase):
   tasks.add_task('regression', testcase.key.id(), testcase.job_type)
 
 
-def create_stack_task_if_needed(testcase):
-  """Creates a stack task if needed."""
-  second_stack_job_type = environment.get_value('SECOND_STACK_JOB_TYPE')
-  if not second_stack_job_type:
-    return
+def create_variant_tasks_if_needed(testcase):
```

Comment:
Should we do any additional filtering on something like APP_NAME?

Reply:
Yes i have to think more and add things. APP_NAME becomes different even for Chrome, it is chrome.exe for win, chrome for linux, and some Chromium.app for mac.